src/api/handlers/webhook.ts

```diff
@@ -192,7 +198,10 @@ export function deleteWebhookMessage(
  * Global commands are cached for **1 hour**. That means that new global commands will fan out slowly across all guilds, and will be guaranteed to be updated in an hour.
  * Guild commands update **instantly**. We recommend you use guild commands for quick testing, and global commands when they're ready for public use.
  */
-export function createSlashCommand(options: CreateSlashCommandOptions) {
+export async function createSlashCommand(
+  options: ApplicationCommand,
+  guildID?: string,
+) {
```

Comment:
Move `guildID` to `options` to avoid breaking changes.

Reply:
But thats against types rewrite

Reply:
Even though this is breaking I would not recommend that.

Reply:
@itohatweb alright